src/ngraph/frontend/onnx_import/op/shrink.hpp

```diff
@@ -29,6 +29,12 @@ namespace ngraph
         {
             namespace set_1
             {
+                /// @brief ONNX Shrink operator
+                ///
+                /// @note It operates on a single input tensor and two attributes: lambd and bias.
+                ///       Input values that fall into the <-lambd;lambd> range are zeroed-out.
```

Comment:
This sentence reads a bit unclearly to me, perhaps we can clear it up like this:

Input values that which are higher than `lambd` or lower than  `-lambd` are zeroed-out.